### PR TITLE
update laravel quickstarts

### PR DESCRIPTION
**Pull request by glena**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/server-apis/php-laravel/authenticate.md [hunk 1]
```diff
@@ -45,23 +46,9 @@ Optionally, if you want to use the [facade](http://laravel.com/docs/facades) cal
...
-If you want to restrict access with the Auth0 Middleware, you will need to add it in `app/Http/Kernel.php`
-
-```php
-...
-
-protected $routeMiddleware = [
-	...
-	'auth0.jwt' => '\Auth0\Login\Middleware\Auth0JWTMiddleware',
-];
+Now, you will be able to access to the logged user info with `Auth0::getUser()`.
```
Comment (chenkie) [inline articles/server-apis/php-laravel/authenticate.md]: `Now, you will be able` => `You will now be able...`

File: articles/server-apis/php-laravel/authenticate.md [hunk 2]
```diff
@@ -98,6 +85,132 @@ RewriteCond %{HTTP:Authorization} ^(.*)
 RewriteRule .* - [e=HTTP_AUTHORIZATION:%1]
 ```
 
+### 5. Defining a user and a user provider
+
+The [Laravel authentication system](https://laravel.com/docs/5.3/authentication) needs a *User Object* given by a *User Provider*. With these two abstractions, the user entity can have any structure you like and can be stored anywhere. You configure the *User Provider* indirectly, by selecting a user provider in `app/config/auth.php`. The default provider is Eloquent, which persists the User model in a database using the ORM.
+
+The plugin comes with an authentication driver called auth0. This driver defines a user structure that wraps the [Normalized User Profile](/user-profile) defined by Auth0, and it doesn't actually persist the object, it just stores it in the session for future calls.
```
Comment (chenkie) [inline articles/server-apis/php-laravel/authenticate.md]: Backticks around auth0. "Driver called `auth0`"

File: articles/server-apis/php-laravel/authenticate.md [hunk 3]
```diff
@@ -98,6 +85,132 @@ RewriteCond %{HTTP:Authorization} ^(.*)
...
+...
+    'providers' => [
+        'users' => [
+            'driver' => 'auth0'
+        ],
+    ],
+...
+```
+
+If you need to implement a more advanced custom solution, you can always extend the Auth0UserRepository (or implement your own) in order to get and update the user data on your database and event more advaced validations.
```
Comment (chenkie) [inline articles/server-apis/php-laravel/authenticate.md]: Backticks around Auth0UserRepository

File: articles/server-apis/php-laravel/authenticate.md [hunk 4]
```diff
@@ -98,6 +85,132 @@ RewriteCond %{HTTP:Authorization} ^(.*)
...
+      }
+
+      return $user;
+    }
+
+    public function getUserByIdentifier($identifier) {
+        //Get the user info of the user logged in (probably in session)
+        $user = \App::make('auth0')->getUser();
+
+        if ($user===null) return null;
```
Comment (chenkie) [inline articles/server-apis/php-laravel/authenticate.md]: Spaces around the === ? Sorry to nitpick :P

File: articles/server-apis/php-laravel/authenticate.md (hunk 1 above)
Comment (glena) [reply articles/server-apis/php-laravel/authenticate.md]: Done

File: articles/server-apis/php-laravel/authenticate.md (hunk 3 above)
Comment (glena) [reply articles/server-apis/php-laravel/authenticate.md]: Done

File: articles/server-apis/php-laravel/authenticate.md (hunk 4 above)
Comment (glena) [reply articles/server-apis/php-laravel/authenticate.md]: Done

File: articles/server-apis/php-laravel/authenticate.md (hunk 2 above)
Comment (glena) [reply articles/server-apis/php-laravel/authenticate.md]: Done

**Pull request merged**